cli/bench/space.go

```diff
@@ -54,3 +57,36 @@ func dropBenchmarkSpace(tarantoolConnection *tarantool.Connection) error {
 	}
 	return nil
 }
+
+// fillBenchmarkSpace fills benchmark space
+// with a PreFillingCount number of records
+// using connectionPool for fast filling.
+func fillBenchmarkSpace(ctx context.BenchCtx, connectionPool []*tarantool.Connection) int {
+	var insertMutex sync.Mutex
+	var waitGroup sync.WaitGroup
+	filledCount := 0
+	for i := 0; i < ctx.Connections; i++ {
+		waitGroup.Add(1)
+		go func(tarantoolConnection *tarantool.Connection) {
+			defer waitGroup.Done()
+			for filledCount < ctx.PreFillingCount {
+				insertMutex.Lock()
```

Comment:
It seems that now the insertion is not paralleled and we waste extra time switching threads (which are idle waiting for the mutex). I.e., do we have any profit from using multiple goroutines?

Reply:
fixed as possible